lesson2/Strings/src/com/company/Main.java

```diff
@@ -0,0 +1,39 @@
+package com.company;
+
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Entrance point to the program.
+ * Check the entered text to the specified rules
+ * @author Vaitsakhouski Raman
+ * @since 08-10-2016
+ */
+public class Main {
+
+    /**
+     * Entrance point to the program
+     */
+    public static void main(String[] args) {
+
+        /**
+         * Methods invocation
+         * @param text - string, in which writting text
+         */
+        String text=null;
+        System.out.println("Write text.");
+        Scanner sc = new Scanner(System.in);
+        if (sc.hasNextLine()) {
+            text=sc.nextLine();
+        }
+        Rule1 vs = new Rule1(text);

```

Comment:
what if you need to add 100 billions of rules? Will you add it one-by-one? Rework and make this piece of code more generic